### DISCUSSION TITLE
【项目组队】Sui telegram 类似微信红包功能 Bounty：1000U

### DISCUSSION THREAD

**Discussion by vladilen11** [creatorsand/.github#33] [Ideas]
项目名称：Sui telegram 类似微信红包功能

项目描述：

1. 群红包
2. 指定人红包
3. 可以选平分还是随机
4. 这个需要后端知识最好是rust 或者node
5. 这个需要自己想玩法

项目发起人：uvd

其他角色需求：后端

有无Bounty：1000U

原链：https://github.com/movefuns/co-learn/discussions/3

**Comment by uvd** [comment]
这个项目很好 需要想好玩法和设计 不能莽做 最终版本可能都需要安全审计

**Comment by Akagi201** [comment]
有兴趣, near 生态有类似的东西

**Reply by fyang1024** [reply]
找到了 https://github.com/Dispa1r/NearLuckyRobot

**Comment by fyang1024** [comment]
几年前有人做过 https://turbowallet.io/， 可是好像烂尾了。我有兴趣做这个，用`node`做过web2的后端，用`rust`做过链
